model/src/lib.rs

```diff
@@ -37,6 +62,48 @@ impl Signature {
             v: bytes[64],
         }
     }
+
+    fn signing_digest_typed_data(
+        &self,
```

Comment:
This and the following function don't use `self` so could remove the `&self` here and then call them through `Signature::signing_...`.

Reply:
Very nice catch!